### PR TITLE
Remove unused shell script

### LINKED ISSUE #926: Fix bash linter warnings in .bashrc_linux64
```console
In environment/bashrc/.bashrc_linux64 line 13:
if [[ -n "$verbose" ]]; then echo "In .bashrc_linux64"; fi
          ^------^ SC2154: verbose is referenced but not assigned.


In environment/bashrc/.bashrc_linux64 line 21:
  target=`uname -n`
         ^--------^ SC2006: Use $(...) notation instead of legacy backticked `...`.

Did you mean: 
  target=$(uname -n)


In environment/bashrc/.bashrc_linux64 line 37:
target=`uname -n`
       ^--------^ SC2006: Use $(...) notation instead of legacy backticked `...`.

Did you mean: 
target=$(uname -n)


In environment/bashrc/.bashrc_linux64 line 48:
      module use --append $HOME/privatemodules
                          ^---^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
      module use --append "$HOME"/privatemodules


In environment/bashrc/.bashrc_linux64 line 54:
    if [[ `groups | grep -c ccsrad` != 0 ]]; then
          ^-----------------------^ SC2006: Use $(...) notation instead of legacy backticked `...`.

Did you mean: 
    if [[ $(groups | grep -c ccsrad) != 0 ]]; then


In environment/bashrc/.bashrc_linux64 line 74:
add_to_path ${DRACO_SRC_DIR}/environment/latex TEXINPUTS
            ^--------------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
add_to_path "${DRACO_SRC_DIR}"/environment/latex TEXINPUTS


In environment/bashrc/.bashrc_linux64 line 75:
add_to_path ${DRACO_SRC_DIR}/environment/bibtex BSTINPUTS
            ^--------------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
add_to_path "${DRACO_SRC_DIR}"/environment/bibtex BSTINPUTS


In environment/bashrc/.bashrc_linux64 line 76:
add_to_path ${DRACO_SRC_DIR}/environment/bibfiles BIBINPUTS
            ^--------------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
add_to_path "${DRACO_SRC_DIR}"/environment/bibfiles BIBINPUTS

For more information:
  https://www.shellcheck.net/wiki/SC2154 -- verbose is referenced but not ass...
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...
  https://www.shellcheck.net/wiki/SC2006 -- Use $(...) notation instead of le...]
2020-10-13 15:58:29 [ERROR ]   Linter CMD:[shellcheck --color --external-sources environment/bashrc/.bashrc_linux64]
2020-10-13 15:58:29 [INFO  ]   ---------------------------
```

### PR DESCRIPTION
**Pull request by KineticTheory**
### Background

* Still trying to get all of the `super-linter` checks to pass

### Purpose of Pull Request

+ Fixes #926

### Description of changes

+ Also make 100-column updates for `.bashrc`

### Status

* Reference:
  * [Pre-Merge Code Review](https://github.com/lanl/Draco/wiki/Style-Guide)
  * [CDash Status](https://rtt.lanl.gov/cdash/index.php?project=Draco&filtercount=1&showfilters=1&field1=buildname&compare1=63&value1=-pr)
* Checks
  * [x] Travis/Appveyor CI checks pass
  * [x] Code coverage does not decrease
  * [x] [Valgrind test passes](https://rtt.lanl.gov/CDash/index.php?project=Draco)
  * [x] [Toss3 checks pass](https://rtt.lanl.gov/CDash/index.php?project=Draco)
  * [x] [Trinitite checks pass](https://rtt.lanl.gov/CDash/index.php?project=Draco)
  * [x] Code reviewed/approved, sufficient DbC checks, testing, documentation

### CODE REVIEW

File: environment/bashrc/.bashrc [hunk 1]
```diff
@@ -217,6 +215,6 @@ if [[ "${verbose:=false}" == "true" ]]; then
   echo "in draco/environment/bashrc/.bashrc ... done";
 fi
 
-##---------------------------------------------------------------------------##
-## end of .bashrc
-##---------------------------------------------------------------------------##
+#---------------------------------------------------------------------------#
+# end of .bashrc
+#---------------------------------------------------------------------------#
```
Comment (keadyk) [inline environment/bashrc/.bashrc:220]: Minor: should these lines also go to 100 columns? :)

File: environment/bashrc/.bashrc [hunk 2]
```diff
@@ -78,18 +76,18 @@ case ${-} in
     esac
     ;; # end case 'interactive'
 
-  ##---------------------------------------------------------------------------##
-  ## ENVIRONMENTS for non interactive sessions
-  ##---------------------------------------------------------------------------##
+  #---------------------------------------------------------------------------#
+  # ENVIRONMENTS for non interactive sessions
+  #---------------------------------------------------------------------------#
```
Comment (keadyk) [inline environment/bashrc/.bashrc:81]: Minor: Should these be updated to 100 columns? :)

Comment (KineticTheory) [reply environment/bashrc/.bashrc]: Fixed by 7b03549.

File: environment/bashrc/.bashrc (hunk 1 above)
Comment (KineticTheory) [reply environment/bashrc/.bashrc]: Fixed by 7b03549

**Comment by keadyk** [comment]
LGTM! Merging now :)

**Pull request merged**